code/segmentedtickerplant/stplog.q

```diff
@@ -28,9 +28,15 @@ logname[`none]:{[dir;tab;logfreq;dailyadj]
 
 // Custom mode
 // Periodic-only mode
-// TO DO - Add tabular-only mode, mixed periodic/tabular mode
+// Tabular-only mode
+// Custom function option
 logname[`custom]:{[dir;tab;logfreq;dailyadj]
-  ` sv(hsym dir;`$"custom",ssr[;;""]/[-13_string logfreq xbar .z.p+dailyadj;":.D"]) 
+  if[.stplg.customlogmode~`period;
+    :` sv(hsym dir;`$"customperiod",ssr[;;""]/[-13_string logfreq xbar .z.p+dailyadj;":.D"])];
```

Comment:
I see magic numbers :eyes: can remove them by casting out the necessary date + time:
```suggestion
    :` sv(hsym dir;`$"customperiod",raze[string"du"$logfreq xbar .z.p]except".:";
```